gammapy/cube/core.py

```diff
@@ -324,6 +324,22 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos, copy=True):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap` along slicepos.
+
+        Parameters
+        ----------
+        slicepos : int
+            Slice position
```

Comment:
I've started to use `idx` for "index" in other places.
So my suggestion here (and in other places for consistency) would be to use this name and description:
```
idx_energy : int
    Energy slice index
```

Also: please document the `copy` option in the docstring. (I know ... it's trivial ... but the simplest is just to always document all parameters without any thought / discussion when to document them or not.)